index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div>
+            <h3 class="logo">My Logo</h3>
+        </div>
+        <div class="menu">
+            <img src="./assets/icons/Union.png" alt="Menu Icon">
+        </div>
+    </header>
+
+    <main>
+        <section class="headline-section">
+            <h1 class="heading-1">I'm Abdul Glad to see you!</h1>
+            <p class="intro-info">I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <a class="connect-info">LET’S CONNECT</a>
+
+            <div class="main-icons">
+                <a href="twitter.com"><img class="logo-images" src="./assets/icons/Twitter_logo.svg" alt="Twitter_logo"></a>
+                <a href="linkedin.com"><img class="logo-images" src="./assets/icons/Linkedin_icon.svg" alt="Linkedin Icon"></a>
+                <a href="medium.com"><img class="logo-images" src="./assets/icons/Medium.svg" alt="Medium logo"></a>
+                <a href="Github.com"><img class="logo-images" src="./assets/icons/Github_icon.svg" alt="Github logo"></a>
+                <a href="angelist.com"><img class="logo-images" src="./assets/icons/Angelist.svg" alt="Angelist logo"></a>
+            </div>
```

Comment:
- [ ] Kindly make sure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). This means that you have to change `div tag` to `ul` and you have to wrap each `a tag` in `li tag`.